### ISSUE TITLE
[E304] false positive insie shell: cmd:

### ISSUE THREAD

**Issue by ssbarnea** [ansible/ansible-lint#481]
# Issue Type
- Bug report

The code below throws a false-positive of `[E304] Environment variables don't work as part of command`:

```
    - shell:
        cmd: echo
        environment: '{{ foo }}'
```

Apparently using the known and document `cmd` arg on `shell` makes the linter believe `command` module was used.

**Comment by awcrosby** [comment]
I am no longer seeing that with v4.1.0 which was released today which includes #477. Let me know if pops up again.
